README.md

```diff
@@ -10,6 +10,31 @@ _COSCRAD_ is a loose collaboration of several organizations, technical teams, an
 
 We have adopted the monorepo approach to allow our members to maximize opportunities for code sharing and collaboration, while maintaining autonomy and focus on their own individual projects within this workspace.
 
+### Projects
+#### Apps
+Our apps are of two kinds:
+**core coscrad apps**
+- `api` (core back-end for the COSCRAD `web of knowledge`)
+- `coscrad-frontend` (standard front-end for the COSCRAD `web of knowledge`)
+
+**community experimental prototypes**
+- `tng-dictionary` (maintained by Justin Bambrick, TNG)
+- `tsilqotin-language-hub` (maintained by Justin Bambrick, TNG)
+- `tng-radio-hub` (maintained by Blake Sellars, TNG)
+
+The core apps adhere to strict static analysis, and leverage relatively comprehensive automated  unit \ integration test coverage (Jest), and e2e tests (Cypress). Meanwhile, the experimental prototypes are more 'Wild West'. Many ideas explored in the prototypes will eventually find their way into Coscrad's core.
```

Comment:
I like 'Wild West' amongst ourselves, but we may not want to imply that the language hub and radio hub are haywire to a public audience.  Maybe just something like "Meanwhile, the experimental prototypes do not have full test coverage and are subject to less rigorous reviews."